### DISCUSSION TITLE
Questions on CSS

### DISCUSSION THREAD

**Discussion by eliahru** [MMRIZE/MMM-CalendarExt3Agenda#11] [Q&A] [answered]
Hi @eouia,

Thank you for the great module! Switched to it from MMM-WeeklySchedule recently, and find it much more easy to edit the classes in a separate calendars rather than editing the configuration every time. :) 

If you don't mind, I would like to ask two questions regarding CSS:
1. Is it possible to make the horizontal line appear only between days, but not before the first one? Because now it has two lines, one below the title, and then the one above the first day.
2. If I need only "today" and "next day having events" to be displayed, if it is possible to limit the amount of agenda blocks displayed via CSS? From your hints I know that it's possible to hide blocks having no events, but in my case it would be useful to show, for example, Friday (today) and Monday (next working day), while skipping empty Saturday and Sunday. But if I set 4 days in settings, during the week it will show me Mon-Tue-Wed-Thu. Hope my explanation makes sense. :)

Thank you in advance!

Regards,
Ilya

**Comment by eouia** [comment, marked as answer]
1.
```css
.CX3A .agenda .cell:nth-of-type(1) {
  border-top: none;
}
```

2. Hmmmm... somehow difficult.
I'll consider it in the next update. You probably want just to set it to display "only days has event(s)" by number.

**Comment by eouia** [comment]
Updated.
See #12
